pkg/cache/context.go

```diff
@@ -663,35 +663,31 @@ func (ctx *Context) IsPodFitNode(name, node string, allocate bool) error {
 	return err
 }
 
-func (ctx *Context) IsPodFitNodeViaPreemption(name, node string, allocations []string, startIndex int) (index int, ok bool) {
+func (ctx *Context) IsPodFitNodeViaPreemption(name, node string, allocations []string, startIndex int) (int, bool) {
 	// assume minimal pods need killing if running in testing mode
 	if ctx.apiProvider.IsTestingMode() {
-		return startIndex, ok
+		return startIndex, false
```

Comment:
This will get removed by #832, so we can revert this change to avoid conflicts later

Reply:
I guess @wilfred-s is going to merge it, so please ignore my comment :)